app/src/main/assets/migrations/27.sql

```diff
@@ -1,3 +0,0 @@
-ALTER TABLE TransmitterData ADD COLUMN filtered_data DOUBLE;
```

Comment:
the one in 31 is to add it to the BgReadings table, we also need it in the transmitter data table (which is what this particular line Im referencing was)